test/e2e/logforwarding/elasticsearchunmanaged/forward_to_unmanaged_elasticsearch_test.go

```diff
@@ -1,4 +1,4 @@
-package logforwarding
+package elasticsearchunmanaged
```

Comment:
side question, but why are we bothering to test something that is unmanaged?

Reply:
its really only a sanity check that we properly wire what you spec in the LF with secrets, etc and the output conf of fluent